### PR TITLE
Issues/mcr 2074 mcrurn granular rest service does not register urn for landing page after urn assignment

### PR DESCRIPTION
**Pull request by chiliego**
[Link to jira](https://mycore.atlassian.net/browse/MCR-).

### CODE REVIEW

File: mycore-user2/src/main/java/org/mycore/user2/MCRUser.java [hunk 1]
```diff
@@ -119,12 +120,12 @@
...
 
     /** The name of the person that this login user represents */
-    @MCRUserAttribute
+    @org.mycore.user2.annotation.MCRUserAttribute
     @XmlElement
     private String realName;
 
     /** The E-Mail address of the person that this login user represents */
-    @MCRUserAttribute
+    @org.mycore.user2.annotation.MCRUserAttribute
```
Comment (adlerre) [inline mycore-user2/src/main/java/org/mycore/user2/MCRUser.java:128]: Codacy Issue found: [Unnecessary use of fully qualified name 'org.mycore.user2.annotation.MCRUserAttribute' due to existing same package import 'org.mycore.user2.*'](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=4998072)

**Comment by adlerre** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 2
- Added 2
           

Complexity increasing per file
==============================
- mycore-user2/src/main/java/org/mycore/user2/MCRUserAttribute.java  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=4998072&bid=16494546)

File: mycore-user2/src/main/java/org/mycore/user2/MCRUser.java [hunk 2]
```diff
@@ -664,7 +668,7 @@ public void setValidUntil(Date validUntil) {
         return Arrays.stream(getRoles()).map(MCRRole::getName).collect(Collectors.toSet());
     }
 
-    @MCRUserAttribute(name = "roles", separator = ";")
+    @org.mycore.user2.annotation.MCRUserAttribute(name = "roles", separator = ";")
```
Comment (adlerre) [inline mycore-user2/src/main/java/org/mycore/user2/MCRUser.java:671]: Codacy Issue found: [Unnecessary use of fully qualified name 'org.mycore.user2.annotation.MCRUserAttribute' due to existing same package import 'org.mycore.user2.*'](https://app.codacy.com/manual/MyCoRe/mycore/pullRequest?prid=4998072)

**Comment by chiliego** [comment]
Wrong branch

**Pull request closed without merging**